helpers/swap/OneInchCall.ts

```diff
@@ -54,3 +54,23 @@ export const getOneInchCall =
       exchangeCalldata: response.tx.data,
     }
   }
+
+export const optimismLiquidityProviders = [
```

Comment:
This is so we can exclude Balancer. Because we use Balancer as Flashloan Provider we cannot use Balancer as a 1inch route without throwing a re-entrancy error.